app/views/profiles/delete.html.erb

```diff
@@ -0,0 +1,38 @@
+<% @title = t('.title') %>
+
+<% unless @only_owner_gems.empty? %>
```

Comment:
Let's try to express this positively - it's a double negative and difficult to understand.